Course/Course/Presentation/Unit/CourseUnitView.swift

```diff
@@ -26,19 +26,45 @@ public struct CourseUnitView: View {
     }
     @State var offsetView: CGFloat = 0
     @State var showDiscussion: Bool = false
-    @Environment(\.presentationMode) private var presentationMode
+    @Environment(\.isPresented) private var isPresented
     @Environment(\.isHorizontal) private var isHorizontal
     private let sectionName: String
     public let playerStateSubject = CurrentValueSubject<VideoPlayerState?, Never>(nil)
     
+    //Dropdown parameters
+    @State var showDropdown: Bool = false
+    private let portraitTopSpacing: CGFloat = 60
+    private let landscapeTopSpacing: CGFloat = 75
+    
+    let isDropdownActive: Bool = {
+        Container.shared.resolve(ConfigProtocol.self)?.uiComponents.isVerticalsMenuEnabled ?? false
```

Comment:
Would be better to pass this as an argument to the initializer or use ConfigProtocol from the viewModel

Reply:
done